projects/demo/src/modules/components/input-slider/input-slider.component.ts

```diff
@@ -132,9 +132,9 @@ export class ExampleTuiInputSliderComponent extends AbstractExampleTuiControl {
 
     customContentSelected = null;
 
-    get secondary(): string | null {
+    get secondary(): string {
         return this.secondarySelected === this.secondaryVariants[0]
             ? `${Math.round((this.control.value / this.max) * 100)}%`
-            : this.secondarySelected;
+            : this.secondarySelected ?? '';
```

Comment:
Let's change to `||` to simplify transpiled code

Reply:
fixed